### PR TITLE
Added the radiation beam renderable

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/OpenRA/ra2/issues/43.

### CODE REVIEW

**Comment by Mailaender** [comment]
I think it was done to avoid rounding errors. https://github.com/AttacqueSuperior/ValiantShades/commit/f51fc3f57e3271027f363d052dbd6787f3edac53 I accidentally copied and pasted it.

File: mods/ra2/weapons/zaps.yaml [hunk 1]
```diff
@@ -170,13 +170,13 @@ SuperComet:
 
 RadBeamWeapon:
 	ValidTargets: Ground
+	InvalidTargets: Structure, Wall, ImmuneToRadiation
```
Comment (MustaphaTR) [inline mods/ra2/weapons/zaps.yaml]: Units that were immune to radiation were still targetable by Desolator in the original game, so move this to warhead.

Comment (MustaphaTR) [reply mods/ra2/weapons/zaps.yaml]: Also Harvesters and Other Dedolators should be immune to rad too.

Comment (Mailaender) [reply mods/ra2/weapons/zaps.yaml]: This will probably cause problems with the AI, but okay.

**Pull request merged**